File: WebApi/DataModels/Score.cs
using System;
using FstgWebApi.DataContracts;

namespace FstgWebApi.DataModels
{
    public class Score: IScore
    {
        public int _id { get; set; }
        public int UserId { get; set; }
        public int Value { get; set; }
    }
}

File: WebUi/src/app/models/score.ts
export interface IBaseScore {
  userId: number;
  value: number;
}

export interface IScore extends IBaseScore {
  _id: string;
}

export class Score implements IScore {
  _id: string;
  userId: number;
  value: number;

  constructor(_id: string = '', userId: number = 0, value: number = 0) {
    this._id = _id;
    this.userId = userId;
    this.value = value;
  }
}
